src/components/DashboardCreate/DashboardCreateContainer.js

```diff
@@ -53,7 +48,7 @@ function DashboardCreateContainer() {
         <Route
           path="song"
           element={
-            <RequireAuth siteMembership={LANGUAGE_ADMIN} withMessage>
+            <RequireAuth siteMembership={ASSISTANT} withMessage>
```

Comment:
Should these be editor for now?

Reply:
I'm not sure. If so, we should change Phrase and Word to Editor as well. 
@caforbes ?

Reply:
I'm not sure what this is for specifically -- the buttons on the create page? -- but Assistants can create songs and stories so this seems fine to me.

Reply:
This file/change specifically is for the the route itself, but this PR also changes the buttons to Assistant for stories on the create page.